### ISSUE TITLE
Update network test setup for new fields

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#527]
Sorry, I should have caught this earlier. The batman mesh setup PR added new fields to wifi setup, so we need to update the network tests setUp.
```
======================================================================
ERROR: test_edit_ethernet_connection (plinth.tests.test_network.TestNetwork)
Check that we can update an ethernet connection.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jvalleroy/fbx/plinth/plinth/tests/test_network.py", line 88, in setUp
    cls.wifi_uuid = network.add_connection(wifi_settings)
  File "/home/jvalleroy/fbx/plinth/plinth/network.py", line 433, in add_connection
    connection = _update_settings(None, connection_uuid, settings)
  File "/home/jvalleroy/fbx/plinth/plinth/network.py", line 422, in _update_settings
    _update_wireless_settings(connection, settings['wireless'])
  File "/home/jvalleroy/fbx/plinth/plinth/network.py", line 386, in _update_wireless_settings
    band = wireless['band'] if wireless['band'] != 'auto' else None
KeyError: 'band'
```

**Comment by nikhilrayaprolu** [comment]
is it enough to update test data by adding new fields in https://github.com/freedombox/Plinth/blob/master/plinth/tests/test_network.py

**Comment by jvalleroy** [comment]
@nikhilrayaprolu Yes I think so. Please test it though. :)

**Comment by jvalleroy** [comment]
Fixed in #530.
